templates/html/templates/common/instance_view_builder.html

```diff
@@ -171,20 +172,20 @@
             {% if conf.debug %}
                 <p class="debug-render">skos editorial part for <strong>{{ instance_uri }}</strong></p>
             {% endif %}
-            {{ macros.render_description_item_plain_literal (content,"definition", "Definition", True) }}
-            {{ macros.render_description_item_plain_literal (content, "scopeNote", "Scope note", True) }}
-            {{ macros.render_description_item_plain_literal (content, "example", "Example", True) }}
-            {{ macros.render_description_item_plain_literal (content, "changeNote", "Change note", True) }}
-            {{ macros.render_description_item_plain_literal (content, "historyNote", "History note", True) }}
-            {{ macros.render_description_item_plain_literal (content, "editorialNote", "Editorial note", True) }}
-            {{ macros.render_description_item_plain_literal (content, "note", "Note" , True) }}
-            {{ macros.render_description_item_local_iri (content, "member", "Members", None,True) }}
-            {{ macros.render_description_item_local_iri (content, "broaderConcept", "Broader Concepts", None,True) }}
+            {{ macros.render_description_item_plain_literal (content,"definition", "Definition", "skos:definition", True, accordion_content_state="show") }}
+            {{ macros.render_description_item_plain_literal (content, "scopeNote", "Scope note", "skos:scopeNote", True, accordion_content_state="show") }}
```

Comment:
in this usage the parameter `accordion_content_state="show"` takes a string as a parameter, while in the one on the next line it is a boolean value `accordion_content_state=True`; 

We seem to need to work on the type consistency here.